hotel_housekeeping/tests/test_hotel_housekeeping.py

```diff
@@ -0,0 +1,84 @@
+# See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import common
+from datetime import datetime
+import time
+
+
+class TestHousekeeping(common.TransactionCase):
+    self.env = self.env(context=dict(
+        self.env.context,
+        'tracking_disable': True,
+    ))
```

Comment:
this should reside inside `setUp()`, AFAICT

Reply:
@arkostyuk Amended changes by 
`context = self.env.context`
`self.env = self.env(context=context.update({'tracking_disable': True}))`

Reply:
yes right let me amend it :)